projects/personachat/kvmemnn/kvmemnn.py

```diff
@@ -579,23 +580,24 @@ def predict(self, xs, ys=None, cands=None, cands_txt=None, obs=None):
                                 pred[i] = -1000
                     val,ind=pred.sort(descending=True)
                     # predict the highest scoring candidate, and return it.
-                    print("   [query:          " + self.v2t(xsq) + "]")
+                    #print("   [query:          " + self.v2t(xsq) + "]")
                     ps = []
                     for c in obs[0]['mem']:
                         ps.append(self.v2t(c))
-                    print("   [persona:        " + '|'.join(ps) + "]")
-                    print("   [1st hop qmatch: " + ypredorig + "]")
-                    print("   [1st hop nextut: " + ypred + "]")
+                    #print("   [persona:        " + '|'.join(ps) + "]")
+                    #print("   [1st hop qmatch: " + ypredorig + "]")
+                    #print("   [1st hop nextut: " + ypred + "]")
                     if self.tricks:
                         ypred = ztxt[ind.data[0]] # match
                         self.cands_done.append(ypred)
                     else:
                         ypred = self.fixedCands_txt[ind.data[0]] # match
                         ypred2 = cands_txt2[0][ind.data[0]] # reply to match
                         self.cands_done.append(ind.data[0])
-                        print("   [2nd hop nextut: " + ypred2 + "]")
+                        #print("   [2nd hop nextut: " + ypred2 + "]")
                     tc = [ypred]
                     self.history['labels'] = [ypred]
+                    #print("   [final pred: " + ypred + "]")
```

Comment:
approved although idk if you want to keep these prints

Reply:
yes, commented on purpose